scale/job/models.py

```diff
@@ -1052,6 +1052,10 @@ def schedule_job_executions(self, framework_id, job_executions, workspaces):
             job_exe.started = started
             job_exe.node = node
             job_exe.configure_docker_params(framework_id, workspaces)
+            
+            for key, value in job_exe.job.job_type.docker_params:
+                job_exe.configuration['job_task']['docker_params'].append({'flag': key, 'value': value})
+                
```

Comment:
For clarrification.  Does this mean that you want us to do the changes you are asking above, or dod you already make the changes?